sdk/src/store.rs

```diff
@@ -1772,8 +1784,12 @@ impl Store {
                 crate::claim::RemoteManifest::NoRemote => dest_path.to_path_buf(),
                 crate::claim::RemoteManifest::SideCar => {
                     // remove any previous c2pa manifest from the asset
-                    remove_jumbf_from_file(dest_path)?;
-                    dest_path.with_extension(MANIFEST_STORE_EXT)
+                    match remove_jumbf_from_file(dest_path) {
```

Comment:
Duplicate logic of what's at line 1755. Could this be consolidated?

Reply:
Not easily since this block is within a feature flag

Reply:
I pushed a collapsed version but harder to read the cases